packages/x-date-pickers/src/models/pickers.ts

```diff
@@ -0,0 +1,10 @@
+import type { PickersShortcutsItem } from '../PickersShortcuts';
+
+export interface PickerChangeHandlerContext<TError> {
+  validationError: TError;
+  /**
+   * Label of the shortcut causing this `onChange` call.
+   * If the call has not been caused by a shortcut selection, it will equal `null`.
+   */
+  shortcut: Omit<PickersShortcutsItem<unknown>, 'getValue'> | null;
```

Comment:
What do you think about creating a separate interface that we would export for this? 🤔 
IMHO, them example using this typing doesn't look DX friendly. 🙈

Reply:
Done :+1: